source/stormwater.rst

```diff
@@ -0,0 +1,329 @@
+.. _stormwater:
+
+***********************************************
+Stormwater Retention Model
```

Comment:
Given that Milan brought it up, let's revisit the model naming question. "Urban Rain Model" or "Urban Water Model" sound good to me, to highlight how it's different from the Urban Flood model. Suggestions welcome! I'm inclined to leave out "Retention" since that is only one of several outputs.

Reply:
The model name is not difficult to change in the code, if we end up changing it.